content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Upgrade with Helm
+description: How to upgrade Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: yes
```

Comment:
This says `yes` which generates the snippets, but I don't see the test.

Reply:
Just need to build some test.sh scripts that call the snips to test them. https://github.com/istio/istio.io/blob/master/tests/README.md

Reply:
or you can make it `no` for the time being and I can pick up the tests . Anyways I was thinking of doing the integration tests, so docs test can help build an understanding. let me know

Reply:
I'll do the basic docs tests since it's mostly copy-paste from the existing ambient install tests, we still need proper integ tests with traffic verification.

Reply:
Done, PTAL.